lib/web/ui/integration.go

```diff
@@ -141,6 +141,10 @@ type AWSOIDCDeployServiceRequest struct {
 	// If deploying a Database Service, you should use the SubnetIDs returned by the List Database API call.
 	SubnetIDs []string `json:"subnetIds"`
 
+	// SecurityGroups to apply to the service's network configuration.
+	// If empty, the default security group for the VPC is going to be used.
+	SecurityGroups []string `json:"securityGroups"`
```

Comment:
Should we add `omitempty` tag on it? Not sure how AWS API behaves if you send an empty list. Is it possible it will not assign any security groups if the field is present in the request but is empty?

Reply:
This is the request the UI sends, so I'm not sure adding an `omitempty` changes anything
We would still get the default value for `[]string`

Also, according to their docs
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ecs@v1.29.6/types#AwsVpcConfiguration
> If you don't specify a security group, the default security group for the VPC is used.

During my tests, not sending the value to the AWS SDK and sending an empty slice resulted in the same output from the dashboard (you can see the `Security Groups` with a `-`